app/api/helpers/ticketing.py

```diff
@@ -167,7 +168,7 @@ def charge_paypal_order_payment(order, paypal_payer_id, paypal_payment_id):
             create_pdf_tickets_for_holder(order)
 
             # send email and notifications
-            send_email_to_attendees(order, order.user_id)
+            send_email_to_attendees(order, current_user.id)
```

Comment:
what changes with this?

Reply:
Didn't know this, will remove this change.

Reply:
Removed. Please check now.